frontend/tests/e2e_tests/run

```diff
@@ -2,16 +2,10 @@
 set -e
 
 # by default just add minio, with COMPOSE_CMD_BASE this creates the standard onprem ST setup
-COMPOSE_CMD="docker compose -p gui-tests \
-             -f ${INTEGRATION_PATH}/docker-compose.yml \
-             -f ${INTEGRATION_PATH}/docker-compose.client.yml \
-             -f ${INTEGRATION_PATH}/docker-compose.demo.yml \
-             -f ${INTEGRATION_PATH}/docker-compose.storage.minio.yml"
-ENTERPRISE_COMPOSE=" -f ${INTEGRATION_PATH}/docker-compose.enterprise.yml"
+COMPOSE_CMD="docker compose -p gui-tests -f ${SERVER_ROOT}/dev/docker-compose.yml"
```

Comment:
The `-p` argument could also be set in the compose file `name` top level attribute.